docs: use helm charts instead of manifests + document KeptnConfig CRD (#747)
keptn/lifecycle-toolkit: docs/content/en/docs/concepts/keptnConfig/_index.md

---

> title: KeptnConfigs

Reviewer: ```suggestion
title: KeptnConfig
```

description: Learn what Keptn Configs are and how to use them
icon: concepts
layout: quickstart
weight: 10
hidechildren: true # this flag hides all sub-pages in the sidebar-multicard.html
---

> ### Keptn Config
> A `KeptnConfig` is a CRD used to define configuration values of the Keptn Lifecycle Toolkit.

Reviewer: ```suggestion
A `KeptnConfig`  CRD defines configuration values for the Keptn Lifecycle Toolkit.
```
No need for a sub-header in such a small page

> In the current state, there is a possibility to configure url of OTel collector.

Reviewer: ```suggestion
Currently, it can be used to configure the URL of the OpenTelemetry collector.
```
